core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -707,6 +714,14 @@ class ZkUtils(val zkClient: ZkClient,
     }
   }
 
+  def getTopicPartitionCount(topic: String, defaultNumPartitions: Int = 50): Int = {
```

Comment:
Should this be `getInternalTopicPartitionCount`. Otherwise, the default seems strange.